app/bundles/LeadBundle/Resources/views/Company/list_rows_contacts.html.twig

```diff
@@ -192,7 +192,9 @@
                     </td>
                     <td>
                         <a href="{{ path('mautic_contact_action', {'objectAction': 'view', 'objectId': contact.id}) }}" data-toggle="ajax">
-                            <div>{% if contact.isAnonymous %}{{ contact.primaryIdentifier|trans }}{% else %}{{ contact.primaryIdentifier|purify }}{% endif %}</div>
+                            {% set primaryIdentifier = contact.isAnonymous ? contact.primaryIdentifier|trans : contact.primaryIdentifier|purify %}
+                            {{ primaryIdentifier }}
+                            <div> {{ customContent('lead.name.after', _context) }}</div>
```

Comment:
can you let `<div>` and other added html code outside of this and keep only {{ customContent('lead.name.after', _context) }} 

You can add HTML inside you custom added content. 
In other templates too. Then it could go